ozaria/site/styles/play/level/level-dialogue-view.sass

```diff
@@ -3,143 +3,59 @@
-  // Bounce in
-  @include transition(1s cubic-bezier(.17,.89,.42,1.36))
-  z-index: 1
-
-  &.active
-    display: block
-    bottom: -20px
-
-  &.debrief
-    bottom: -300px
-
-  &.active.debrief
-    bottom: -65px;
-
-  &.speaking
-    .dialogue-area
-      .bubble
-        @include animation(speakingPulse 1.5s infinite)
+  left: 0
+  top: 0
+  width: 56.5%
+  height: 145px
+  background: transparentize(black, 0.6)
+  z-index: 22
 
   .dialogue-area
+    display: flex
     position: relative
-    height: 100%
+    justify-content: center
+    align-items: center
+    height: 106px
     width: 100%
-    z-index: 1
-
-    .bubble
-      position: relative
-      margin: 10px
-      padding: 20px 20px 40px 20px
-      color: white
-      font-weight: bold
-      background: transparent url(/images/level/dialogue_background.png)
-      background-size: 100% 100%
-      border: black solid 1px
-      border-radius: 10px
-      font-size: 18px
-      line-height: 20px
-
-      strong, a
-        color: #FFCCAA
-
-      a
-        text-decoration: underline
-
-      .hud-hint
-        font-weight: normal
-        color: #ddd
-        font-size: 14px
-        line-height: 16px
-        vertical-align: middle
-
-      .enter
-        position: absolute
-        right: 15px
-        bottom: 20px
-        div.dot
-          background: #337
-          width: 8px
-          height: 8px
-          position: absolute
-          right: 8px
-          top: 9px
-          border-radius: 5px
-
-      button, .alert
-        padding: 2px 5px
-
-      .enter button.with-dot
-        padding-right: 20px
-
-      h3
-        margin: 0
-        font-size: 16px
-        line-height: 16px
-        color: #338
-
-      button
-        margin-left: 10px
-  
-  &.debrief
-    z-index: 11
-    width: 100vw
-    max-width: 1024px
-    position: absolute
-    left: 50vw
-    transform: translateX(-50%)
-    img:not(.embiggen)
-      position: absolute
-      width: 20%
-      bottom: 25%
-      filter: drop-shadow(8px 8px 8px black)
-      left: 2.5%
-    .embiggen
+    border: 4px solid #1FBAB4
+    background-color: #FFFFFF
+    box-shadow: 2px 2px 8px 0 #ACB9FC
+    margin-top: 20px
+    margin-bottom: 20px
+    margin-left: 30px
+    margin-right: 30px
+
+    .vega-narrator
       position: absolute
+      width: 14.5%
+      width: 134px
       left: 0
-      width: 25%
-      bottom: 0
-      filter: drop-shadow(8px 8px 8px black)
-    .bubble
-      border: initial
-      width: 75%
-      height: 66.66%
-      right: 16px
-      position: absolute
-      top: 50%
-      transform: translateY(-66.66%)
-      padding: 64px !important
-    @media (max-width: 800px)
-      height: 50%
-      .bubble > p
-        margin-top: -32px
-
-body.ipad #level-dialogue-view
-  left: auto
-  right: 0
-
-  &.active
-    bottom: -60px
+      top: -1px
+      height: 100px
 
-  .hud-hint
-    visibility: hidden
+    .vega-dialogue
+      color: #232323
```

Comment:
Could it be advantageous for some of the values we are using be stored as variables in an `ozaria-style-variables.sass` file?